### PR TITLE
RYA-255 Provide an example class that demonstrates the Prospector.

### PR DESCRIPTION
**Pull request by kchilton2**

### CODE REVIEW

**Comment by pujav65** [comment]
hey @kchilton2 can you fix the conflict?  then i will merge.

File: extras/indexingExample/src/main/java/ProspectorExample.java [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+        log.info("Running the Map Reduce job that computes the Prospector results.");
+        ToolRunner.run(new Prospector(), new String[]{ "src/main/resources/stats_cluster_config.xml" });
+
+        // Print the table that was created by the Prospector.
+        log.info("");
+        log.info("The following cardinalities were written to the Prospector table:");
+        final ProspectorServiceEvalStatsDAO dao = ProspectorServiceEvalStatsDAO.make(config);
+
+        // Do each of the Subjects.
+        double cardinality = dao.getCardinality(config, CARDINALITY_OF.SUBJECT, Lists.newArrayList(ALICE));
```
Comment (meiercaleb) [inline extras/indexingExample/src/main/java/ProspectorExample.java]: Eric's comment seems very valid to me.

Comment (kchilton2) [inline extras/indexingExample/src/main/java/ProspectorExample.java]: I think reducing the lines of the code, in this case, actually makes the example a lot harder to understand.

**Comment by kchilton2** [comment]
Once RYA-254 is pulled in, I'll update this pull request.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-rya-master-with-optionals-pull-requests/499/

**Pull request closed without merging**